### ISSUE TITLE
shopping-cart-java not working for maven in Dev Mode

### ISSUE THREAD

**Issue by CianSteenkamp96** [lagom/lagom-samples#59]
When running `mvn lagom:runAll` instead of 'sbt runAll' (after following the other necessary instructions in the readme) the project starts successfully but the curl commands do not work. That is when running any of the curl commands I get an error. The error is: 
```bash
<!DOCTYPE html>
<html lang="en">
    <head>
        ...
    </head>
    <body>
        <h1>Action Not Found</h1>

        <p id="detail">
            For request 'GET /shoppingcart/123'
        </p>

        

                <h2>
                    These routes have been tried, in this order:
                </h2>

                <div>
                    
                </div>

            

    </body>
</html>
```

**Comment by ignasi35** [comment]
Hi @CianSteenkamp96 , thanks for reporting this. I've tried to reproduce but everything seems to work as expected on my machine.

```

ignasi at imslightbend in ~/git/projects/lightbend/lagom/lagom-samples on zk-three-replicas*
$ cd shopping-cart/shopping-cart-java/
ignasi at imslightbend in ~/git/projects/lightbend/lagom/lagom-samples/shopping-cart/shopping-cart-java on zk-three-replicas*
$ docker-compose up -d
Pulling postgres (postgres:latest)...
latest: Pulling from library/postgres
743f2d6c1f65: Already exists
5d307000f290: Pull complete
29837b5e9b78: Pull complete
3090df574038: Pull complete
dc0b4463fa0e: Pull complete
1fb834895f59: Pull complete
59169bd605be: Pull complete
a950d631bfe9: Pull complete
7db7e3522bac: Pull complete
a4bc301cd166: Pull complete
3c8614737f74: Pull complete
ad575a1ac336: Pull complete
a6431194f884: Pull complete
a4501fbec177: Pull complete
Creating shopping_cart_postgres ... done
ignasi at imslightbend in ~/git/projects/lightbend/lagom/lagom-samples/shopping-cart/shopping-cart-java on zk-three-replicas*
$ mvn lagom:runAll
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] shopping-cart-java                                                 [pom]
[INFO] shopping-cart-api                                                  [jar]
[INFO] shopping-cart                                                      [jar]
[INFO] inventory-api                                                      [jar]
[INFO] inventory                                                          [jar]
[INFO]
[INFO] -------------------< com.example:shopping-cart-java >-------------------
[INFO] Building shopping-cart-java 1.0-SNAPSHOT                           [1/5]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- lagom-maven-plugin:1.5.1:runAll (default-cli) @ shopping-cart-java ---
[INFO] Starting Kafka
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
2019-05-24T07:45:04.124Z [info] akka.event.slf4j.Slf4jLogger [] - Slf4jLogger started
2019-05-24T07:45:06.312Z [warn] akka.actor.ActorSystemImpl [sourceThread=main, akkaSource=akka.actor.ActorSystemImpl(application), sourceActorSystem=application, akkaTimestamp=07:45:06.310UTC] - Detected that Hostname Verification is disabled globally (via ssl-config's akka.ssl-config.loose.disableHostnameVerification) for the Http extension! This is very dangerous and may expose you to man-in-the-middle attacks. If you are forced to interact with a server that is behaving such that you must disable this setting, please disable it for a given connection instead, by configuring a specific HttpsConnectionContext for use only for the trusted target that hostname verification would have blocked.
2019-05-24T07:45:07.546Z [info] com.lightbend.lagom.registry.impl.ServiceLocatorServer [] - Service locator can be reached at 127.0.0.1
2019-05-24T07:45:07.546Z [info] com.lightbend.lagom.registry.impl.ServiceLocatorServer [] - Service gateway can be reached at 127.0.0.1
[INFO] Service locator is running at http://127.0.0.1:9008
[INFO] Service gateway is running at http://127.0.0.1:9000
09:45:10.274 [info] play.core.server.AkkaHttpServer [] - Listening for HTTP on /127.0.0.1:49696
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/ignasi/git/projects/lightbend/lagom/lagom-samples/shopping-cart/shopping-cart-java/shopping-cart-api/src/main/resources
[INFO] Nothing to compile - all classes are up to date
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO] Nothing to compile - all classes are up to date
09:45:13.550 [info] akka.event.slf4j.Slf4jLogger [] - Slf4jLogger started
09:45:13.595 [info] akka.remote.Remoting [sourceThread=main, akkaSource=akka.remote.Remoting, sourceActorSystem=shopping-cart-application, akkaTimestamp=07:45:13.593UTC] - Starting remoting
09:45:13.789 [info] akka.remote.Remoting [sourceThread=main, akkaTimestamp=07:45:13.789UTC, akkaSource=akka.remote.Remoting, sourceActorSystem=shopping-cart-application] - Remoting started; listening on addresses :[akka.tcp://shopping-cart-application@127.0.0.1:58613]
09:45:13.793 [info] akka.remote.Remoting [sourceThread=main, akkaTimestamp=07:45:13.792UTC, akkaSource=akka.remote.Remoting, sourceActorSystem=shopping-cart-application] - Remoting now listens on addresses: [akka.tcp://shopping-cart-application@127.0.0.1:58613]
09:45:13.819 [info] akka.cluster.Cluster(akka://shopping-cart-application) [sourceThread=main, akkaTimestamp=07:45:13.819UTC, akkaSource=akka.cluster.Cluster(akka://shopping-cart-application), sourceActorSystem=shopping-cart-application] - Cluster Node [akka.tcp://shopping-cart-application@127.0.0.1:58613] - Starting up, Akka version [2.5.22] ...
09:45:13.945 [info] akka.cluster.Cluster(akka://shopping-cart-application) [sourceThread=main, akkaTimestamp=07:45:13.945UTC, akkaSource=akka.cluster.Cluster(akka://shopping-cart-application), sourceActorSystem=shopping-cart-application] - Cluster Node [akka.tcp://shopping-cart-application@127.0.0.1:58613] - Registered cluster JMX MBean [akka:type=Cluster,port=58613]
09:45:13.946 [info] akka.cluster.Cluster(akka://shopping-cart-application) [sourceThread=main, akkaTimestamp=07:45:13.945UTC, akkaSource=akka.cluster.Cluster(akka://shopping-cart-application), sourceActorSystem=shopping-cart-application] - Cluster Node [akka.tcp://shopping-cart-application@127.0.0.1:58613] - Started up successfully
09:45:13.973 [info] akka.cluster.Cluster(akka://shopping-cart-application) [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-2, akkaSource=akka.cluster.Cluster(akka://shopping-cart-application), sourceActorSystem=shopping-cart-application, akkaTimestamp=07:45:13.973UTC] - Cluster Node [akka.tcp://shopping-cart-application@127.0.0.1:58613] - No seed-nodes configured, manual cluster join required, see https://doc.akka.io/docs/akka/current/cluster-usage.html#joining-to-seed-nodes
09:45:14.586 [info] play.api.db.DefaultDBApi [] - Database [default] initialized at jdbc:postgresql://localhost/shopping_cart
09:45:14.597 [info] application [] - Creating Pool for datasource 'default'
09:45:14.608 [info] com.zaxxer.hikari.HikariDataSource [] - HikariPool-1 - Starting...
09:45:14.622 [info] com.zaxxer.hikari.HikariDataSource [] - HikariPool-1 - Start completed.
09:45:14.634 [info] play.api.db.HikariCPConnectionPool [] - datasource [default] bound to JNDI as DefaultDS
09:45:15.405 [info] akka.cluster.Cluster(akka://shopping-cart-application) [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-5, akkaSource=akka.cluster.Cluster(akka://shopping-cart-application), sourceActorSystem=shopping-cart-application, akkaTimestamp=07:45:15.404UTC] - Cluster Node [akka.tcp://shopping-cart-application@127.0.0.1:58613] - Node [akka.tcp://shopping-cart-application@127.0.0.1:58613] is JOINING itself (with roles [dc-default]) and forming new cluster
09:45:15.413 [info] akka.cluster.Cluster(akka://shopping-cart-application) [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-5, akkaTimestamp=07:45:15.411UTC, akkaSource=akka.cluster.Cluster(akka://shopping-cart-application), sourceActorSystem=shopping-cart-application] - Cluster Node [akka.tcp://shopping-cart-application@127.0.0.1:58613] - is the new leader among reachable nodes (more leaders may exist)
09:45:15.426 [info] akka.cluster.Cluster(akka://shopping-cart-application) [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-5, akkaSource=akka.cluster.Cluster(akka://shopping-cart-application), sourceActorSystem=shopping-cart-application, akkaTimestamp=07:45:15.425UTC] - Cluster Node [akka.tcp://shopping-cart-application@127.0.0.1:58613] - Leader is moving node [akka.tcp://shopping-cart-application@127.0.0.1:58613] to [Up]
09:45:15.499 [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-2, akkaTimestamp=07:45:15.499UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/system/sharding/ShoppingCartEntityCoordinator, sourceActorSystem=shopping-cart-application] - Singleton manager starting singleton actor [akka://shopping-cart-application/system/sharding/ShoppingCartEntityCoordinator/singleton]
09:45:15.513 [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-2, akkaTimestamp=07:45:15.513UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/system/sharding/ShoppingCartEntityCoordinator, sourceActorSystem=shopping-cart-application] - ClusterSingletonManager state change [Start -> Oldest]
09:45:15.520 [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-15, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/jdbcCreateTables-singleton, sourceActorSystem=shopping-cart-application, akkaTimestamp=07:45:15.520UTC] - Singleton manager starting singleton actor [akka://shopping-cart-application/user/jdbcCreateTables-singleton/singleton]
09:45:15.521 [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-15, akkaTimestamp=07:45:15.520UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/jdbcCreateTables-singleton, sourceActorSystem=shopping-cart-application] - ClusterSingletonManager state change [Start -> Oldest]
09:45:15.549 [info] com.lightbend.lagom.internal.persistence.cluster.ClusterStartupTaskActor [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-15, akkaTimestamp=07:45:15.548UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/jdbcCreateTables-singleton/singleton/jdbcCreateTables, sourceActorSystem=shopping-cart-application] - Executing cluster start task jdbcCreateTables.
09:45:15.556 [info] akka.cluster.sharding.DDataShardCoordinator [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-18, akkaTimestamp=07:45:15.556UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/system/sharding/ShoppingCartEntityCoordinator/singleton/coordinator, sourceActorSystem=shopping-cart-application] - ShardCoordinator was moved to the active state State(Map(),Map(),Set(),Set(),false)
09:45:15.642 [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-4, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/slickOffsetStorePrepare-singleton, sourceActorSystem=shopping-cart-application, akkaTimestamp=07:45:15.642UTC] - Singleton manager starting singleton actor [akka://shopping-cart-application/user/slickOffsetStorePrepare-singleton/singleton]
09:45:15.643 [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-4, akkaTimestamp=07:45:15.642UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/slickOffsetStorePrepare-singleton, sourceActorSystem=shopping-cart-application] - ClusterSingletonManager state change [Start -> Oldest]
09:45:15.643 [info] com.lightbend.lagom.internal.persistence.cluster.ClusterStartupTaskActor [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-16, akkaTimestamp=07:45:15.642UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/slickOffsetStorePrepare-singleton/singleton/slickOffsetStorePrepare, sourceActorSystem=shopping-cart-application] - Executing cluster start task slickOffsetStorePrepare.
09:45:15.717 [info] com.lightbend.lagom.internal.persistence.cluster.ClusterStartupTaskActor [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-16, akkaTimestamp=07:45:15.716UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/slickOffsetStorePrepare-singleton/singleton/slickOffsetStorePrepare, sourceActorSystem=shopping-cart-application] - Cluster start task slickOffsetStorePrepare done.
09:45:15.737 [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-17, akkaTimestamp=07:45:15.737UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/system/sharding/kafkaProducer-shopping-cartCoordinator, sourceActorSystem=shopping-cart-application] - Singleton manager starting singleton actor [akka://shopping-cart-application/system/sharding/kafkaProducer-shopping-cartCoordinator/singleton]
09:45:15.738 [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-17, akkaTimestamp=07:45:15.737UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/system/sharding/kafkaProducer-shopping-cartCoordinator, sourceActorSystem=shopping-cart-application] - ClusterSingletonManager state change [Start -> Oldest]
09:45:15.738 [info] akka.cluster.sharding.DDataShardCoordinator [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-17, akkaTimestamp=07:45:15.737UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/system/sharding/kafkaProducer-shopping-cartCoordinator/singleton/coordinator, sourceActorSystem=shopping-cart-application] - ShardCoordinator was moved to the active state State(Map(),Map(),Set(),Set(),false)
09:45:15.782 [info] com.lightbend.lagom.internal.persistence.cluster.ClusterStartupTaskActor [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-4, akkaTimestamp=07:45:15.782UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/jdbcCreateTables-singleton/singleton/jdbcCreateTables, sourceActorSystem=shopping-cart-application] - Cluster start task jdbcCreateTables done.
09:45:15.866 [info] play.api.http.EnabledFilters [] - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):


09:45:15.977 [info] play.api.Play [] - Application started (Dev) (no global state)
09:45:16.466 [info] akka.cluster.singleton.ClusterSingletonProxy [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-18, akkaTimestamp=07:45:16.466UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/jdbcCreateTables-singletonProxy, sourceActorSystem=shopping-cart-application] - Singleton identified at [akka://shopping-cart-application/user/jdbcCreateTables-singleton/singleton]
09:45:16.648 [info] akka.cluster.singleton.ClusterSingletonProxy [sourceThread=shopping-cart-application-akka.actor.default-dispatcher-15, akkaTimestamp=07:45:16.647UTC, akkaSource=akka.tcp://shopping-cart-application@127.0.0.1:58613/user/slickOffsetStorePrepare-singletonProxy, sourceActorSystem=shopping-cart-application] - Singleton identified at [akka://shopping-cart-application/user/slickOffsetStorePrepare-singleton/singleton]
09:45:18.203 [info] play.core.server.AkkaHttpServer [] - Listening for HTTP on /127.0.0.1:56768
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/ignasi/git/projects/lightbend/lagom/lagom-samples/shopping-cart/shopping-cart-java/shopping-cart-api/src/main/resources
[INFO] Nothing to compile - all classes are up to date
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/ignasi/git/projects/lightbend/lagom/lagom-samples/shopping-cart/shopping-cart-java/inventory-api/src/main/resources
[INFO] Nothing to compile - all classes are up to date
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO] Nothing to compile - all classes are up to date
09:45:20.321 [info] akka.event.slf4j.Slf4jLogger [] - Slf4jLogger started
09:45:21.221 [info] play.api.http.EnabledFilters [] - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):


09:45:21.268 [info] play.api.Play [] - Application started (Dev) (no global state)
[INFO] Service shopping-cart listening for HTTP on 127.0.0.1:49696
[INFO] Service inventory listening for HTTP on 127.0.0.1:56768
[INFO] (Services started, press enter to stop and go back to the console...)
09:45:22.029 [warn] org.apache.kafka.clients.NetworkClient [] - [Consumer clientId=inventory-1, groupId=inventory] Error while fetching metadata with correlation id 2 : {shopping-cart=LEADER_NOT_AVAILABLE}

```

**Comment by octonato** [comment]
Moreover, we are testing it against an OpenShift cluster each time code is merged. So it's known to work.

@CianSteenkamp96, please check your environment. There must be something locally.

**Comment by CianSteenkamp96** [comment]
@ignasi35 and @renatocaval Thank you for the quick response. Here is what is shown on my side when running `mvn lagom:runAll`

```bash
/shopping-cart/shopping-cart-java$ docker-compose up -d
shopping_cart_postgres is up-to-date
/shopping-cart/shopping-cart-java$ mvn lagom:runAll
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] shopping-cart-java                                                 [pom]
[INFO] shopping-cart-api                                                  [jar]
[INFO] shopping-cart                                                      [jar]
[INFO] inventory-api                                                      [jar]
[INFO] inventory                                                          [jar]
[INFO] 
[INFO] -------------------< com.example:shopping-cart-java >-------------------
[INFO] Building shopping-cart-java 1.0-SNAPSHOT                           [1/5]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- lagom-maven-plugin:1.5.1:runAll (default-cli) @ shopping-cart-java ---
[INFO] Starting Kafka
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
2019-05-24T09:03:12.872Z [info] akka.event.slf4j.Slf4jLogger [] - Slf4jLogger started
2019-05-24T09:03:14.300Z [warn] akka.actor.ActorSystemImpl [sourceThread=main, akkaSource=akka.actor.ActorSystemImpl(application), sourceActorSystem=application, akkaTimestamp=09:03:14.299UTC] - Detected that Hostname Verification is disabled globally (via ssl-config's akka.ssl-config.loose.disableHostnameVerification) for the Http extension! This is very dangerous and may expose you to man-in-the-middle attacks. If you are forced to interact with a server that is behaving such that you must disable this setting, please disable it for a given connection instead, by configuring a specific HttpsConnectionContext for use only for the trusted target that hostname verification would have blocked.
2019-05-24T09:03:14.699Z [info] com.lightbend.lagom.registry.impl.ServiceLocatorServer [] - Service locator can be reached at 127.0.0.1
2019-05-24T09:03:14.700Z [info] com.lightbend.lagom.registry.impl.ServiceLocatorServer [] - Service gateway can be reached at 127.0.0.1
[INFO] Service locator is running at http://127.0.0.1:9008
[INFO] Service gateway is running at http://127.0.0.1:9000
[WARN] [05/24/2019 11:03:15.876] [main] [ManifestInfo(akka://shopping-cart-internal-dev-mode)] Detected possible incompatible versions on the classpath. Please note that a given Akka HTTP version MUST be the same across all modules of Akka HTTP that you are using, e.g. if you use [10.1.8] all other modules that are released together MUST be of the same version. Make sure you're using a compatible set of libraries. Possibly conflicting versions [10.1.8, 10.1.7] in libraries [akka-http-spray-json:10.1.8, akka-parsing:10.1.7, akka-http:10.1.8, akka-http-core:10.1.7]
11:03:16.228 [info] play.core.server.AkkaHttpServer [] - Listening for HTTP on /127.0.0.1:49696
[WARN] [05/24/2019 11:03:17.357] [main] [ManifestInfo(akka://inventory-internal-dev-mode)] Detected possible incompatible versions on the classpath. Please note that a given Akka HTTP version MUST be the same across all modules of Akka HTTP that you are using, e.g. if you use [10.1.8] all other modules that are released together MUST be of the same version. Make sure you're using a compatible set of libraries. Possibly conflicting versions [10.1.8, 10.1.7] in libraries [akka-http-spray-json:10.1.8, akka-parsing:10.1.7, akka-http:10.1.8, akka-http-core:10.1.7]
11:03:17.698 [info] play.core.server.AkkaHttpServer [] - Listening for HTTP on /127.0.0.1:56768
[INFO] Service shopping-cart listening for HTTP on 127.0.0.1:49696
[INFO] Service inventory listening for HTTP on 127.0.0.1:56768
[INFO] (Services started, press enter to stop and go back to the console...)
```

**Comment by CianSteenkamp96** [comment]
Any idea what can be missing on my side? I followed the readme exactly. As mentioned I only get the error once executing a curl command such as `curl http://localhost:9000/shoppingcart/123`.

**Comment by CianSteenkamp96** [comment]
If you prefer to continue this discussion via email here is my email address: ciansteenkamp@gmail.com

**Comment by ignasi35** [comment]
Hi @CianSteenkamp96, the output you shared is suspiciously short (see the output I shared).

What versions of java and maven are you using?

```
$ java -version
openjdk version "1.8.0_202"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_202-b08)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.202-b08, mixed mode)
ignasi at imslightbend in ~/git/personal/apache
$ mvn --version
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T20:41:47+02:00)
Maven home: /usr/local/Cellar/maven/3.6.0/libexec
Java version: 1.8.0_202, vendor: AdoptOpenJdk, runtime: /Library/Java/JavaVirtualMachines/jdk8u202-b08_hotspot/Contents/Home/jre
Default locale: en_ES, platform encoding: UTF-8
OS name: "mac os x", version: "10.12.6", arch: "x86_64", family: "mac"
```

**Comment by octonato** [comment]
@CianSteenkamp96, better not to switch to email because others facing the same issue can learn how to solve it.

**Comment by ignasi35** [comment]
> Moreover, we are testing it against an OpenShift cluster each time code is merged. So it's known to work.

@renatocaval the CI jobs in travis don't exercise the dev mode of the samples.

**Comment by CianSteenkamp96** [comment]
@ignasi35 my java and maven version:

```bash
cian@cian:~$ java -version
java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)
cian@cian:~$ mvn --version
Apache Maven 3.6.0
Maven home: /usr/share/maven
Java version: 1.8.0_201, vendor: Oracle Corporation, runtime: /usr/lib/jvm/java-8-oracle/jre
Default locale: en_ZA, platform encoding: UTF-8
OS name: "linux", version: "4.15.0-50-generic", arch: "amd64", family: "unix"
```

**Comment by ignasi35** [comment]
@CianSteenkamp96 I've updated the subject to better detail the issue.

**Comment by CianSteenkamp96** [comment]
@ignasi35 thank you for updating the subject detail. See my maven and java version above. It's the same. I agree, my output is much shorter than yours...

**Comment by CianSteenkamp96** [comment]
> @CianSteenkamp96, better not to switch to email because others facing the same issue can learn how to solve it.

@renatocaval sure thing.

**Comment by ignasi35** [comment]
@CianSteenkamp96 do you have some [global maven plugin](https://stackoverflow.com/questions/26012661/maven-global-plugin-configuration) that may be interfering?

Do you have access to a pristine environment where you can retest?

**Comment by CianSteenkamp96** [comment]
@ignasi35 I do have a settings.xml file in my ~/.m2 folder which I added/created - after removing it, I still have the same problem when running `mvn clean lagom:runAll`. Also, in /etc/maven there is also a settings.xml file (as referred to in the link in your previous message) but I did not create this file nor did I change anything inside it and most of it (except some tags <..>) are commented out. Even after removing this file, as well, I still have the same issue.

Also, I do not have access to a pristine environment.

Thanks for your support thus far.

**Comment by CianSteenkamp96** [comment]
@ignasi35 I also made sure that I don't have my local postgresql service running before starting the docker container.

**Comment by CianSteenkamp96** [comment]
@ignasi35 and @renatocaval I deleted and recloned the git repo. Everything seems to be working now. I apologize for wasting your time. Also, just for clarity, it works even with my settings.xml file in ~/.m2 and the default settings.xml file in /etc/maven. Thanks for the help. I don't know what changed from before since I am 99% sure I did not change anything after downloading/cloning the git initially.
